cli/formatting.go

```diff
@@ -0,0 +1,192 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	eos "github.com/eoscanada/eos-go"
+	"github.com/ryanuber/columnize"
+)
+
+func FormatPermissions(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"permissions:",
+	}
+	for _, perm := range account.Permissions {
+		keysValues := []string{}
+		for _, key := range perm.RequiredAuth.Keys {
+			keysValues = append(keysValues, fmt.Sprintf("%d %s", key.Weight, key.PublicKey))
+		}
+		output = append(output,
+			fmt.Sprintf("     %s |%d:|%s",
+				perm.PermName,
+				perm.RequiredAuth.Threshold,
+				strings.Join(keysValues, ", "),
+			),
+		)
+	}
+	return columnize.Format(output, config)
+}
+
+func FormatMemory(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"memory:",
+		fmt.Sprintf("     quota: %s| used: %s",
+			prettifyBytes(account.RAMQuota),
+			prettifyBytes(account.RAMUsage),
+		),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatNetworkBandwidth(account *eos.AccountResp, config *columnize.Config) string {
+	delegatedNet := account.TotalResources.NetWeight.Sub(account.SelfDelegatedBandwidth.NetWeight)
+
+	output := []string{
+		"net bandwidth:",
+		fmt.Sprintf("     staked:|%s|(total stake delegated from account to self)",
+			prettifyAsset(account.SelfDelegatedBandwidth.NetWeight),
+		),
+		fmt.Sprintf("     delegated:|%s|(total stake delegated to account from others)",

```

Comment:
The CPU part is using `delegated from account` while here is using `delegated to account`. Should probably be the same in both. I just don't know which one is the right wording.

Reply:
I'll pick `to account` and update.

Reply:
Updated